addons/website/static/src/snippets/s_map/options.js

```diff
@@ -84,10 +54,46 @@ options.registry.Map = options.Class.extend({
         if (methodName === 'showDescription') {
             return !!this.$target[0].querySelector('.description');
         }
-        if (methodName === 'setHeight') {
-            return this.$target[0].dataset.mapMinHeight;
-        }
         return this._super(...arguments);
     },
+
+    /**
+     * @override
+     * @private
+     */
+    _computeWidgetVisibility: function (widgetName, params) {
+        if (widgetName === 'default_height') {
+            // Hide if half or full height
+            return !(this.$target.hasClass("o_half_screen_height") || this.$target.hasClass("o_full_screen_height"));
```

Comment:
Can't remember, why are we keeping the height option in "auto" height mode?

Reply:
This was kept because the padding adjustment felt awkward - starting from no padding.
I now replaced the `min-height` by pt+pb and it behaves nicely, also when the map is inside a column.